db/import_test.go

```diff
@@ -79,6 +79,71 @@ func TestFeedImport(t *testing.T) {
 	}
 }
 
+func TestOnDemandImportWithSpecialChars(t *testing.T) {
+	base.SetUpTestLogging(t, base.LevelDebug, base.KeyHTTP, base.KeyCRUD, base.KeyImport)
+	base.SkipImportTestsIfNotEnabled(t)
+
+	db, ctx := SetupTestDBWithOptions(t, DatabaseContextOptions{})
+	defer db.Close(ctx)
+
+	// RFC 4627 2.5
+	// > All Unicode characters may be placed within the
+	// > quotation marks except for the characters that _must_ be escaped:
+	// > quotation mark, reverse solidus, and the control characters (U+0000
+	// > through U+001F).
+	// > Any character _may_ be escaped.
+	tests := []struct {
+		name        string
+		docBody     string
+		expectError bool
+	}{
+		{name: "question", docBody: `{"question":"?"}`},
+		{name: "exclamation", docBody: `{"exclamation":"!"}`},
+		{name: "interrobang", docBody: `{"interrobang":"‽"}`},
+		{name: "single_quote", docBody: `{"single_quote":"'"}`},
+		{name: "ampersand", docBody: `{"ampersand":"&"}`},
```

Comment:
TODO:
- Test JSON Keys
- Test 3.0.x and 2.8.x to determine whether this is a regression
- Find the code that causes this escaping